subsys/suit/orchestrator/src/suit_orchestrator_sdfw.c

```diff
@@ -384,6 +384,10 @@ int suit_orchestrator_init(void)
 			LOG_ERR("Failed to load MPI: invalid MPI format (i.e. version, values)");
 			plat_err = suit_execution_mode_set(EXECUTION_MODE_FAIL_MPI_INVALID);
 			break;
+		case SUIT_PLAT_ERR_EXISTS:
```

Comment:
Probable documentation for suit_plat_err_t suit_storage_init(void) should be amended, especially range of returned error codes.

Reply:
Nice catch!
Documentation extended.